astropy/modeling/tabular.py

```diff
@@ -120,8 +124,33 @@ def __init__(self, points=None, lookup_table=None, method='linear',
 
     @property
     def points(self):
+        """
+        An iterable of points defining the values in the lookup table.
+        """
         return self._points
 
+    @property
+    def bounding_box(self):
+        """
+        Tuple defining the default ``bounding_box`` limits,
+        ``(points_low, points_high)``.
+
+        Examples
```

Comment:
Just want to make sure: The example in this docstring is tested, right? Otherwise, we need to add a test for this.